source/MapPanel.cpp

```diff
@@ -1130,12 +1155,49 @@ void MapPanel::DrawSystems()
 	if(commodity == SHOW_GOVERNMENT)
 		closeGovernments.clear();
 
-	// Draw the circles for the systems.
+
```

Comment:
The changes to 1160-1161 could be reverted and this comment changed to draw rings or star sprites